packages/sanity/src/core/form/components/formField/FormFieldValidationStatus.tsx

```diff
@@ -156,25 +130,68 @@ function FormFieldValidationStatusItem(props: {validation: FormNodeValidation})
 }
 
 function FormFieldValidationSummary({validation}: {validation: FormNodeValidation[]}) {
-  const errorMarkers = validation.filter((item) => item.level === 'error')
-  const warningMarkers = validation.filter((item) => item.level === 'warning')
-  const errorLen = errorMarkers.length
-  const warningLen = warningMarkers.length
+  const {t} = useTranslation()
+  const listFormatter = useIntlListFormat()
+
+  const errorCount = validation.reduce(
+    (count, item) => (item.level === 'error' ? count + 1 : count),
+    0,
+  )
+  const warningCount = validation.reduce(
+    (count, item) => (item.level === 'warning' ? count + 1 : count),
+    0,
+  )
+
+  const hasErrors = errorCount > 0
+  const hasWarnings = warningCount > 0
 
-  const errorsStr = `error${errorLen === 1 ? '' : 's'}`
-  const warningsStr = `warning${warningLen === 1 ? '' : 's'}`
-  const errorText = errorLen && `${errorLen} ${errorsStr}`
-  const warningText = warningLen && `${warningLen} ${warningsStr}`
+  if (!hasErrors && !hasWarnings) {
+    return null
+  }
 
-  const hasErrors = errorLen > 0
-  const hasWarnings = warningLen > 0
-  const hasBoth = hasErrors && hasWarnings
+  const errorText = hasErrors && t('form.validation.summary.errors-count', {count: errorCount})
+  const warningText =
+    hasWarnings && t('form.validation.summary.warnings-count', {count: warningCount})
+
+  return errorText && warningText ? (
+    <Text size={1}>{listFormatter.format([errorText, warningText])}</Text>
+  ) : (
+    <Text size={1}>{errorText || warningText}</Text>
+  )
+}
 
+function ValidationErrorIcon() {
+  const {t} = useTranslation()
   return (
-    <Text size={1}>
-      {errorText || ''}
-      {hasBoth && <> and </>}
-      {warningText || ''}
-    </Text>
+    <ErrorOutlineIcon
+      data-testid="input-validation-icon-error"
+      aria-label={t('form.validation.has-error-aria-label')}
```

Comment:
this is a comment on the old code but i'm unsure if an aria-label is necessary if there is `aria-hidden` and `role="presentation"` on there

Reply:
agreed, but probably something we should look at later when we are focusing on accessibility